doc/_themes/mantella/static/style.css

```diff
@@ -212,7 +212,7 @@ main details pre {
   color: #f8f8f8;
 }
 
-main details > div:nth-child(even) pre {
+main details > div:nth-child(3n) pre {
```

Comment:
[`doc/.preprocessing.py`](https://github.com/Mantella/Mantella/blob/pschiffmann/issue342/doc/.pre_processing.py#L224) inserts the language labels on the same level as the code blocks.

before:
```html
<details>
  <summary>Code example</summary>
  <div class="highlight-c++">...</div>
  <div class="highlight-octave">...</div>
</details>
```

after:
```html
<details>
  <summary>Code example</summary>
  <div class="language"><span>C++</span></div>'
  <div class="highlight-c++">...</div>
  <div class="language"><span>Octave</span></div>'
  <div class="highlight-octave">...</div>
</details>
```

Must be `nth-child(4n - 1)` though.